### ISSUE TITLE
GeoQA Test Set

### ISSUE THREAD

**Issue by eternal8080** [pipilurj/G-LLaVA#19]
I am curious about where the GeoQA test set comes from, because I see that the GeoQA author did not release a separate test set, but mixed the training set and the test set together.
